target/linux/lantiq/files-4.14/arch/mips/boot/dts/FRITZ7412.dts

```diff
@@ -0,0 +1,241 @@
+/dts-v1/;
+
+#include "vr9.dtsi"
+
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/mips/lantiq_rcu_gphy.h>
+
+/ {
+	compatible = "avm,fritz7412", "lantiq,xway", "lantiq,vr9";
+	model = "1&1 DSL-Modem / 1&1 WLAN-Router";
```

Comment:
Name of the Device should be "AVM FRITZ!Box 7412".

The Device might be exclusively available thru 1&1 but the backside of the device reads the AVM naming.

(I know this is very inconsistent in the lantiq target)